src/libsystemd-network/radv-internal.h

```diff
@@ -25,6 +25,16 @@
 #include "list.h"
 #include "sparse-endian.h"
 
+#define SD_RADV_DEFAULT_MIN_TIMEOUT_USEC        (200*USEC_PER_SEC)
+#define SD_RADV_DEFAULT_MAX_TIMEOUT_USEC        (600*USEC_PER_SEC)
+assert_cc(SD_RADV_DEFAULT_MIN_TIMEOUT_USEC <= SD_RADV_DEFAULT_MAX_TIMEOUT_USEC)
+
+#define SD_RADV_MAX_INITIAL_RTR_ADVERT_INTERVAL_USEC (16*USEC_PER_SEC)
+#define SD_RADV_MAX_INITIAL_RTR_ADVERTISEMENTS  3
+#define SD_RADV_MAX_FINAL_RTR_ADVERTISEMENTS    3
+#define SD_RADV_MIN_DELAY_BETWEEN_RAS           3
+#define SD_RADV_MAX_RA_DELAY_TIME_USEC          (0.5*USEC_PER_SEC)
```

Comment:
uh, this macro resolves to a float this way (because 0.5 is a float and not an integfer). please write this like this instead:

```c
#define SD_RADV_MAX_RA_DELAY_TIME_USEC (USEC_PER_SEC/2)
````
or even:

```c
#define SD_RADV_MAX_RA_DELAY_TIME_USEC (500*USEC_PER_MSEC)
```